stack.yaml

```diff
@@ -1,28 +1,24 @@
-resolver: lts-16.31
+resolver: lts-17.14
 
 packages:
 - '.'
 
 extra-deps:
 - git: https://github.com/dapphub/dapptools.git
-  commit: 6314bc8e168fce17290a73700a1246f549e2e052
+  commit: 4fe19a0ff493f09de58c13c54cb69d962dd10b82
   subdirs:
   - src/hevm
 
-- cryptonite-0.27
-- ghci-pretty-0.0.2
-- HSH-2.1.3
-- monoidal-containers-0.6.0.1
-- restless-git-0.7
-- s-cargot-0.1.4.0
-- text-format-0.3.2
-- these-1.0.1
-- tree-view-0.5
-- sbv-8.9
-- witherable-0.3.5
-- witherable-class-0
-- github: dmjio/semver-range
-  commit: d8d9db892ddb6ae267c9bcbc4f6602668433f12a
+- base16-bytestring-1.0.1.0@sha256:33b9d57afa334d06485033e930c6b13fc760baf88fd8f715ae2f9a4b46e19a54,2641
+- restless-git-0.7@sha256:346a5775a586f07ecb291036a8d3016c3484ccdc188b574bcdec0a82c12db293,968
+- s-cargot-0.1.4.0@sha256:61ea1833fbb4c80d93577144870e449d2007d311c34d74252850bb48aa8c31fb,3525
+- sbv-8.15@sha256:bc0ea4e3564626030ac2cb21905b97cd9bd25fddcd6d6010834e8cc2ebf79067,29016
+- semver-range-0.2.8@sha256:44918080c220cf67b6e7c8ad16f01f3cfe1ac69d4f72e528e84d566348bb23c3,1941
+- tree-view-0.5.1@sha256:a32b16fdbe24ad21cb6679c2a925bcc7715ae5c5db0ae6b633e35e28e2fefd98,1184
+- witherable-0.4.1@sha256:5b4840efe20d16cab6b13fa2f39ed383703cccbc246f5b710591069fcc6884c0,2249
+- HSH-2.1.3@sha256:71ded11b224f5066373ce985ec63b10c87129850b33916736dd64fa2bea9ea0a,1705
+- indexed-traversable-instances-0.1@sha256:3aaf97040001bbe583e29c2b9c7d41660df265e6565a0d2ac09a3ed5b8bc21be,2874
+- libBF-0.6.2@sha256:7bffc0e4dbc9bd9e851ba5c29255b62fd2c288dbc9a401cd3761b740f013775e,1770
```

Comment:
do we need the sha hashes?

Reply:
this is what stack suggests, I guess it's the a feature